locales/en/data-classes.ftl

```diff
@@ -34,6 +34,7 @@ customer-interactions = Customer interactions
 dates-of-birth = Dates of birth
 deceased-date = Deceased date
 deceased-statuses = Deceased statuses
+delivery-instructions = Delivery instructions
```

Comment:
What is this? Delivery instructions for drivers (e.g. on Amazon)?

Can we have a comment to give context to localizers?

Reply:
https://monitor.firefox.com/breach-details/Weee is all I know. Let me see if I can fetch the description from HIBP site.

Reply:
> _"In February 2023, *data belonging to the Asian and Hispanic food delivery service Weee appeared on a popular hacking forum*. Dating back to mid-2022, the data included 1.1M unique email addresses from 11M rows of orders containing names, phone numbers and delivery instructions."_

Reply:
OK. Then something like

```
# Instructions that you can set up in online shopping services to get your orders 
# delivered (e.g. deliver to a specific neighbor, entrance location, etc.). 
```